File: servers/dotnet/Application/Users/DTOs/UserDto.cs
using System;
using Application.Blogs.DTOs;

namespace Application.Users.DTOs;

public class UserDto
{
    public required string Id { get; set; }
    public required string Email { get; set; }
    public required string DisplayName { get; set; }
    public string? Description { get; set; }
}

File: servers/springboot/api/src/main/java/com/gabrielgavrilov/ohmyblog/api/UserDto.java
package com.gabrielgavrilov.ohmyblog.api;

import lombok.Data;
import lombok.experimental.Accessors;

import java.util.UUID;

@Data
@Accessors(chain = true)
public class UserDto {
    private UUID userId;
    private String email;
    private String displayName;
    private String description;
}
